packages/opal-server/opal_server/config.py

```diff
@@ -317,4 +329,4 @@ def on_load(self):
             self.SERVER_BIND_PORT = int(self.SERVER_PORT)
 
 
-opal_server_config = OpalServerConfig(prefix="OPAL_")
+opal_server_config = OpalServerConfig(prefix="OPAL_")
```

Comment:
Because this sets the prefix, I can't figure out how to pass in environment variables that don't have this prefix.